src/pages/docs/projects/project-triggers/external-feed-triggers.md

```diff
@@ -0,0 +1,77 @@
+
+The details of these container images and Helm charts are already known in Octopus. This means we can use the registry locations, image names, chart names, and credentials to do this monitoring, without adding or maintaining this information anywhere else.
+
+## Getting started {#ExternalFeedTriggers-GettingStarted}
+
+Navigate to your project’s triggers page by selecting Projects and clicking on the project you are working with. Click **Triggers** option on the left, under **Deployments**.  You are now presented with an empty triggers list.  
+
+Click the **ADD TRIGGER** on the right-hand side of the page, and select **External feed**
+
+:::figure
+![Project triggers list](/docs/projects/project-triggers/images/add-trigger-popup.png)
+:::
+
+Enter a name and description for your trigger.  The name should be short, memorable, and unique. Example: Create a release when nginx is updated in Docker.
+
+## Channels and lifecycles
+
+If your project contains multiple [channels](/docs/releases/channels), you have the option of selecting which channel this trigger will apply to.  Any pushed packages must satisfy the selected channel's [versioning rules](/docs/releases/channels#version-rules) to trigger a release creation. The releases created by the trigger will use this channel.
+
+Unlike the existing [Automatic Release Creation](/docs/projects/project-triggers/automatic-release-creation) feature, you can create multiple external feed triggers per channel.  This can enable you to, for example, have your external container dependencies in a separate trigger to that used for your internal container dependencies.
+
+:::figure
+![Channel selection](/docs/projects/project-triggers/images/external-trigger-channel.png)
+:::
+
+A preview of the [lifecycle](/docs/releases/lifecycles) is also shown, to show how releases created by the trigger will be deployed.
+
+If you combine external feed triggers with the automatic deployment feature within [Lifecycles phases](/docs/releases/lifecycles/#Lifecycles-LifecyclePhases), whenever a package is pushed to a selected external feed, the trigger can create a release, and have it be automatically deployed.
+
+## Trigger sources
+
+Any container images or helm charts referenced in your project's deployment process can be selected to trigger release creation.  Feeds or packages referenced using variable substitution are also options here, however they will only be used by the trigger if they are both evaluated as either container/helm chart repositories, and do not use variable unavailable at release creation time.  For example, using an environment name variable will not work, because that value is only available at deploy time.
```

Comment:
nit about some of the text.

But also, the [writing docs](https://www.octopus.design/932c0f1a9/p/843fd5-documentation/t/43f893) seem to suggest that the 'edge case' of using variable feed types should go in an 'advance use case' section at the end.

```suggestion
Any container images or helm charts referenced in your project's deployment process can be selected to trigger release creation.  Feeds or packages referenced using variable substitution are also options here, however they will only be used by the trigger if they are both evaluated as either container or helm chart repositories, and do not use a variable unavailable at release creation time.  For example, using an environment name variable will not work, because that value is only available at deploy time.
```